imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -385,10 +337,16 @@ def populate_full_cycle_data(
     # data. But for now, we are advice to continue with current setup and can
     # add/change it when we get real data.
 
+    # Apply calibration based on in-flight calibration.
+    corrected_counts = apply_in_flight_calibration(full_cycle_data, acquisition_times)
```

Comment:
Are there multiple "corrections"? Maybe call this explicitly `calibrated_counts`?
```suggestion
    corrected_counts = apply_in_flight_calibration(full_cycle_data, acquisition_times)
```